tests/fixtures/app_store_tests/core/roots.yml

```diff
@@ -0,0 +1,5 @@
+setup_project_root:
```

Comment:
The unit test that required this roots.yml used to write it programmatically. Now it's just part of the config on disk.